changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 26th June 2019 19:00 UTC
+
+* Added [Get all tax enivonments](operations/configuration.md#get-all-tax-environments) operation.
+* Extended [Currency value](operations/finance.md#currency-value) with `TaxValues`.
```

Comment:
Currency value will just contains:
```
Currency,
Value
```
2) There will be new object  AmountParameters. Because it is not extended just TaxValues, but also with GrossValue and NetValue

Reply:
@vitasamek Wouldn't it make sense to have this in the same but just make value optional to be returned?

Reply:
It was discussed in person, still not updated.
A currency value is updated, AmountParameters and Amount added.